packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -18,6 +18,8 @@ import { CameraSwitch24Filled } from '@fluentui/react-icons';
 import { ComponentIcons, DEFAULT_COMPONENT_ICONS } from '@internal/react-components';
 import React from 'react';
 import { mergeStyles, Text } from '@fluentui/react';
+/* @conditional-compile-remove-from(stable) */
```

Comment:
For all these conditional compilation directives, it's a good idea to use tags to group them (so we can find them easily later).

I've been using

```ts
/* conditional-compile-remove-from(stable) meeting-composite <further-details> */
```

for meeting related stuff.